README.md

```diff
@@ -145,6 +145,7 @@ Table of Contents
 ## Cloud management solutions
   * [Brainboard](https://www.brainboard.co) - Collaborative solution to visually build and manage cloud infrastructures from end-to-end.
   * [terraform.io](https://www.terraform.io/) — Terraform Cloud. Free remote state management and team collaboration for teams up to 5 users.
+  * [Meteor Cloud](https://www.meteor.com/cloud) — Galaxy hosting. Meteor's own platform-as-a-service for Meteor apps which includes a free MongoDB Shared Hosting and automatic SSL.
```

Comment:
this category is for tools to build other infrastructure - terraform to build your own AWS instances etc.

So Meteor Cloud might be better elsewhere I think